audmath/core/api.py

```diff
@@ -86,6 +86,183 @@ def db(
     return x
 
 
+def duration_in_seconds(
+        time: typing.Union[float, int, str, np.timedelta64],
```

Comment:
I would also rename the argument to `'duration'`.

Reply:
I changed all remaining `time` variable names to `duration`.